src/main/java/mate/academy/model/Order.java

```diff
@@ -0,0 +1,90 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Objects;
+import javax.persistence.CascadeType;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true)
+    @JoinColumn(name = "order_id")
```

Comment:
What will be the column name for tickets id? Have a look in workbench.

Reply:
agree with @IlkoGood . And add name for new columns. And for columns tickets, orderDate. I also recommend removing cascades